### PR TITLE
[EventEngine] Simple iOS EventEngine experiment configuration.

### PR DESCRIPTION
**Pull request by drfloob**
cc @sampajano

### CODE REVIEW

**Review by sampajano — APPROVED** [review]
@drfloob Thanks so much for helping with the test setup, AJ!!

@HannahShiSFB FYI 😃

File: tools/internal_ci/macos/pull_request/grpc_bazel_cpp_ios_event_engine_experiment_tests.cfg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 The gRPC Authors
```
Comment (yashykt) [inline tools/internal_ci/macos/pull_request/grpc_bazel_cpp_ios_event_engine_experiment_tests.cfg:1]: 2023

Comment (drfloob) [reply tools/internal_ci/macos/pull_request/grpc_bazel_cpp_ios_event_engine_experiment_tests.cfg:1]: image

Comment (drfloob) [reply tools/internal_ci/macos/pull_request/grpc_bazel_cpp_ios_event_engine_experiment_tests.cfg]: fixed

**Pull request merged**